packages/framework/fluid-framework/src/index.ts

```diff
@@ -22,84 +22,16 @@ export type {
 	IFluidContainer,
 	IFluidContainerEvents,
 	IMember,
-	IRootDataObject,
 	IServiceAudience,
 	IServiceAudienceEvents,
 	LoadableObjectClass,
 	LoadableObjectClassRecord,
 	LoadableObjectCtor,
-	LoadableObjectRecord,
```

Comment:
My fix to preserve the typing of 'initialObjects' had the side effect of making this internal.